.npmignore

```diff
@@ -0,0 +1,12 @@
+# unrelated folders
+src/
```

Comment:
wouldn’t it be useful to have the `"module": "src/index"` in package.json so bundlers like rollup can require and tree shake the source code?

Reply:
Yeap, adding a new issue for this. I might move the bundling to microbundle that creates a umd, cjs and module file all at once.